src/test/java/pages/Header.java

```diff
@@ -0,0 +1,26 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+
+public class Header extends BasePage {
+
+    private By searchField = By.className("fast-search__input");
```

Comment:
Please use PageFactory

Reply:
I was faced with the fact that the declared lazy initialization of objects in PageFactory does not work correctly. That's why I didn't use it.